control-plane/pkg/reconciler/source/v2/sourcev2.go

```diff
@@ -101,6 +101,11 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, ks *sources.Kafk
 		},
 		Spec: internalscg.ConsumerGroupSpec{
 			Template: internalscg.ConsumerTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"eventing.knative.dev/sourceUID": string(ks.UID),
```

Comment:
nit: can we use `kafka.eventing.knative.dev/source.uid` ? can we extract it as constant?

should we add a label for `cg` to the trigger reconciler too? maybe in a different PR?

Reply:
@pierDipi sounds good. I added as constant[ here ](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1920/files#diff-678e94010cb81bf07cc8bf28ed64cc0c94c44541d7398c96f2f63296bf0ac7cfR28)

Label for trigger reconciler is already in this PR. I'll update the PR title to reflect that. Is that ok?